sklearn/ensemble/gradient_boosting.py

```diff
@@ -271,8 +273,8 @@ def __call__(self, y, pred, sample_weight=None):
             return (1.0 / sample_weight.sum() *
                     np.sum(sample_weight * ((y - pred.ravel()) ** 2.0)))
 
-    def negative_gradient(self, y, pred, **kargs):
-        return y - pred.ravel()
+    def negative_gradient(self, y, pred, sample_weight=None):
+        return 2.0 * (y - pred.ravel())
```

Comment:
Sample weight ignored in all cases except the Huber loss. I would support phasing it out as a parameter.